src/ui/common/src/components/integrations/addIntegrations.tsx

```diff
@@ -96,6 +96,18 @@ const AddIntegrationListItem: React.FC<AddIntegrationListItemProps> = ({
   const service = svc as Service;
   const [showDialog, setShowDialog] = useState(false);
 
+  // If this is a conda integration, check if it has already been registered.
+  // If it has, disable the new Conda button.
+  const resources = useSelector(
+    (state: RootState) => state.integrationsReducer.integrations
+  );
+  if (svc === 'Conda') {
```

Comment:
Probably not something that needs to be fixed now, but I wonder if there is a functin we can add to SupportedIntegrations enum that allows for us to get this info from the conda dialog.